src/main/java/Lesson5/Exercise2/Game.java

```diff
@@ -0,0 +1,19 @@
+package Lesson5.Exercise2;
+
+public class Game {
+
+    public static void game (String[] args) {
+
+        for (int i = 1; i <= 100; i++) {
+            if (i % 3 == 0 && i % 5 != 0) {
```

Comment:
rozbij to na metody np. 
isDivisibleBy(67, 5);
public static boolean isDivisibleBy(int number, int dividor){
		return number % dividor == 0;
	}